res/battle/graphic/pl_batt_obj/order.txt

```diff
@@ -0,0 +1,343 @@
+battle_terrain_giratina_player.ncgr
+battle_terrain_giratina_opponent.ncgr
+pl_batt_obj_00000177.nanr
+pl_batt_obj_00000178.ncer
+pl_batt_obj_00000179.ncgr
+pl_batt_obj_00000180.nanr
+pl_batt_obj_00000181.ncer
+pl_batt_obj_00000182.ncgr
+pl_batt_obj_00000183.nanr
+pl_batt_obj_00000184.ncer
+pl_batt_obj_00000185.ncgr
+healthbox_singles_opponent.nanr
+healthbox_singles_opponent.ncer
+healthbox_singles_opponent.ncgr
+healthbox_singles_player.nanr
+healthbox_singles_player.ncer
+healthbox_singles_player.ncgr
+healthbox_doubles_opponent_1.nanr
+healthbox_doubles_opponent_1.ncer
+healthbox_doubles_opponent_1.ncgr
+healthbox_doubles_opponent_2.nanr
+healthbox_doubles_opponent_2.ncer
+healthbox_doubles_opponent_2.ncgr
+healthbox_doubles_player_1.nanr
+healthbox_doubles_player_1.ncer
+healthbox_doubles_player_1.ncgr
+healthbox_doubles_player_2.nanr
+healthbox_doubles_player_2.ncer
+healthbox_doubles_player_2.ncgr
+ball_status_indicator_opponent.ncer
+ball_status_indicator_opponent.ncgr
+ball_status_indicator_opponent.nanr
+ball_status_indicator_player.ncer
+ball_status_indicator_player.ncgr
+ball_status_indicator_player.nanr
+pl_batt_obj_00000210.ncer
+pl_batt_obj_00000211.ncgr
+pl_batt_obj_00000212.nanr
+pl_batt_obj_00000213.ncer
+pl_batt_obj_00000214.ncgr
+pl_batt_obj_00000215.nanr
+pl_batt_obj_00000216.ncer
+pl_batt_obj_00000217.ncgr
+pl_batt_obj_00000218.nanr
+contest_icon_beauty.ncgr
+contest_icon_cute.ncgr
+type_icon_dragon.ncgr
+type_icon_electric.ncgr
+type_icon_psychic.ncgr
+type_icon_dark.ncgr
+type_icon_fighting.ncgr
+type_icon_fire.ncgr
+type_icon_flying.ncgr
+type_icon_ghost.ncgr
+type_icon_ground.ncgr
+type_icon_ice.ncgr
+type_icon_bug.ncgr
+contest_icon_smart.ncgr
+type_icon_grass.ncgr
+type_icon_normal.ncgr
+type_icon_poison.ncgr
+type_icon_mystery.ncgr
+type_icon_rock.ncgr
+type_icon_steel.ncgr
+contest_icon_tough.ncgr
+contest_icon_cool.ncgr
+type_icon_water.ncgr
+pl_batt_obj_00000242.ncer
+pl_batt_obj_00000243.nanr
+move_class_physical.ncgr
+move_class_status.ncgr
+move_class_special.ncgr
+pl_batt_obj_00000247.ncgr
+pl_batt_obj_00000248.ncer
+pl_batt_obj_00000249.nanr
+battle_subscreen_cursor_corner.ncgr
+battle_subscreen_cursor_corner.ncer
+battle_subscreen_cursor_corner.nanr
+healthbox_safari.ncgr
+healthbox_safari.ncer
+healthbox_safari.nanr
+bg_party_level_up.ncgr
+bg_party_level_up.ncer
+bg_party_level_up.nanr
+player_lucas_back.ncgr
+player_dawn_back.ncgr
+player_barry_back.ncgr
+player_cheryl_back.ncgr
+player_riley_back.ncgr
+player_marley_back.ncgr
+player_buck_back.ncgr
+player_mira_back.ncgr
+pl_batt_obj_00000267.ncer
+pl_batt_obj_00000268.ncer
+pl_batt_obj_00000269.ncer
+pl_batt_obj_00000270.ncer
+pl_batt_obj_00000271.ncer
+pl_batt_obj_00000272.ncer
+pl_batt_obj_00000273.ncer
+pl_batt_obj_00000274.ncer
+pl_batt_obj_00000275.nanr
+pl_batt_obj_00000276.nanr
+pl_batt_obj_00000277.nanr
+pl_batt_obj_00000278.nanr
+pl_batt_obj_00000279.nanr
+pl_batt_obj_00000280.nanr
+pl_batt_obj_00000281.nanr
+pl_batt_obj_00000282.nanr
```

Comment:
these are almost certainly corresponding with the `player_..._back` sprites, but have not found where they are used in the code to confirm